README.md

```diff
@@ -1,3 +1,19 @@
 # ImageQuiz
 
 See [the documentation website](http://jasig.github.io/ImageQuiz).
+
+## Building
+
+Build using 
+
+```shell_session
```

Comment:
Ah. I was relying upon [the Rouge lexers](https://github.com/jneen/rouge/wiki/List-of-supported-languages-and-lexers) [supported by GitHub Pages](https://help.github.com/articles/using-syntax-highlighting-on-github-pages/), hadn't realized it's a different set for GitHub itself.